system/gatekeeper/tests/doop-analyzer-rules/run.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+set -eou pipefail
+
+# This test validates the processing_rules and merging_rules in templates/configmap-doop-analyzer.yaml by running
+# `doop-analyzer process-once` using that config on the input files at */input.json and checking that the expected
+# output at */output.json is obtained. The test case directories are named after the rules that they test.
+#
+# To build a new test case, you can obtain a suitable input.json from a real cluster by grabbing its doop-analyzer
+# config (`k -n kube-system get cm doop-analyzer`), adding the `kubernetes:` section for use with kubectl credentials
+# (see doop-analyzer README for details), and then running `doop-analyzer collect-once config.yaml > input.json`.
+# Truncate and redact the result as required.
+
+cd "$(dirname "$0")"
+
+CONFIG_FILE=../rendered-chart/gatekeeper/doop-analyzer-config.yaml
+yq -r '.data["config.yaml"]' < ../rendered-chart/gatekeeper/templates/configmap-doop-analyzer.yaml > "$CONFIG_FILE"
```

Comment:
```suggestion
yq -r '.data["config.yaml"]' ../rendered-chart/gatekeeper/templates/configmap-doop-analyzer.yaml > "$CONFIG_FILE"
```
this should also work, right?